PrebidMobile/AdUnits/RewardedVideoAdUnit.swift

```diff
@@ -15,21 +15,24 @@ limitations under the License.
 
 import UIKit
 
-public class RewardedVideoAdUnit: VideoBaseAdUnit {
+@objcMembers
```

Comment:
Just curious - is it mean that this class, and interstitial ad unit as well, hasn't been used in Obj-c apps?

Reply:
Actually, there is no need for this attribute in these classes. They inherit from AdUnit class, which already provides this attribute that also applies to subclasses. Removed it from AdUnit subclasses.